kubernetes/cmsweb/services/reqmgr2ms-transferor.yaml

```diff
@@ -0,0 +1,117 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: reqmgr2ms-t
+  namespace: dmwm
+spec:
+  selector:
+    app: reqmgr2ms-t
+  ports:
+    - port: 8248
```

Comment:
Same port problem here, please update to: https://github.com/dmwm/deployment/blob/master/reqmgr2ms/config-transferor.py#L36